### PR TITLE
GCS_Common: brackets missing in switch case

### PR DESCRIPTION
**Pull request by olliw42**
title says it. Would cause issues if more is to come.

### CODE REVIEW

**Review by magicrub — APPROVED** [review]
LGTM, Thanks @olliw42 !

**Comment by peterbarker** [comment]
@olliw42 thanks - will save someone a little bit of pain in the future, I think.

Marginally contrary to our style guide as the brace should be on the same line as the `case` label.

Note that we don't always require the braces, but where we're introducing a variable like this we would.

**Comment by olliw42** [comment]
> Marginally contrary to our style guide as the brace should be on the same line as the `case` label.

I followed the style above and below that line :) (just scroll down 10 lines or so)

In foreign code I generally watch out for the pattern, but the code style is not consistent across all ArduPilot code, so it's generally difficult to determine the style by looking for the pattern. I thus generally look for the pattern near to the code change. I wouldn't know what better to do. The preferred style also appears to depend on the particular developper. Maybe some clang thing or similar would help you make the style consistent. A more complete style guide could also help; this specific bracket rule is e.g. not specified there :)

The bottom line: Even when paying attention to style (which I do) it's hard to make it correct and avoid respective comments.

> Note that we don't always require the braces, but where we're introducing a variable like this we would.

I know I know ;)

(technically it's not you who requires the braces, right, but the compiler :D )
